fedimint-cli/src/ng.rs

```diff
@@ -103,29 +112,24 @@ pub async fn handle_ng_command<D: IDatabase>(
             amount,
             description,
             expiry_time,
+            wait_for_payment,
         } => {
             let active_gateway = client.fetch_active_gateway().await?;
 
             let (operation_id, _) = client
-                .create_bolt11_invoice_and_receive(amount, description, expiry_time, active_gateway)
+                .create_bolt11_invoice_and_receive(
+                    amount,
+                    description,
+                    expiry_time,
+                    active_gateway,
+                    wait_for_payment,
+                )
```

Comment:
* Rename `create_bolt11_invoice_and_receive` to `create_bolt11_invoice`
* `create_bolt11_invoice` always waits for offer acceptance or rejection internally
* remove `wait_for_payment` argument, never wait for payment

Reply:
Done